ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/sflow.schema.yml

```diff
@@ -51,14 +57,31 @@ keys:
           keys:
             destination:
               type: str
-              description: Sflow Destination IP
+              description: Sflow Destination IP Address
             port:
               type: int
               description: Port Number
               convert_types:
               - str
+      source:
+        type: str
+        description: Source IP Address
```

Comment:
Please make this a multiline string, and add the comment about the two options being mutually exclusive.

Reply:
info added to the description!